### PR TITLE
graveler changes to support ui import

### LINKED ISSUE #3149: Graveler to support new import/ingest commands
Part of #3137 

Graveler needs to support new commands to enable ingest from the ui:
1. Create range directly from an iterator of entries
2. Create metarange directly from a set of ranges
3. Create a commit given a branch and a predefined metarange id.

### PR DESCRIPTION
**Pull request by itaiad200**
closes #3149 

Missing:
1. ~~Unit tests (also system-tests?)~~
2. ~~Figuring out the proper commit action to support.~~ See discussion in #3137 -> No new functionality is needed

### CODE REVIEW

File: pkg/graveler/graveler.go [hunk 1]
```diff
@@ -100,14 +100,36 @@ const (
...
 
-type MetaRangeInfo struct {
-	// URI of metarange file.
-	Address string
-}
+// URI of metarange file.
+type MetaRangeAddress string
 
+// URI of range file.
+type RangeAddress string
```
[the remark this answers is not in the document]
Comment (itaiad200) [reply pkg/graveler/graveler.go]: Done.

File: pkg/graveler/graveler.go [hunk 2]
```diff
@@ -374,9 +396,16 @@ type VersionController interface {
...
 
-	// WriteMetaRange accepts a ValueIterator and writes the entire iterator to a new MetaRange
+	// WriteRange creates a new Range from the iterator values.
+	// Keeps Range closing logic, so might not flush all values to the range.
+	WriteRange(ctx context.Context, repositoryID RepositoryID, it ValueIterator) (*RangeInfo, error)
+
+	// WriteMetaRange creates a new MetaRange from the given Ranges.
+	WriteMetaRange(ctx context.Context, repositoryID RepositoryID, ranges []*RangeInfo) (*MetaRangeInfo, error)
+
+	// FlushToMetaRange accepts a ValueIterator and writes the entire iterator to a new MetaRange
```
[the remark this answers is not in the document]
Comment (itaiad200) [reply pkg/graveler/graveler.go]: Done.

**Review by nopcoder — APPROVED** [review]
lgtm!

**Comment by itaiad200** [comment]
Merging with force due to known issues with GC tests #3189

**Pull request merged**